projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -16,23 +16,50 @@ Import the component in your module:
 
 and in your HTML:
 
-~~~
+~~~html
 <button lg-button type="button" [variant]="solid-primary">Button</button>
 ~~~
 
 or:
 
-~~~
+~~~html
 <a lg-button href="#" [variant]="primary">Link</a>
 ~~~
 
+## Button with icon with text
+
+Icons can be content projected into the button, the button component will then handle the styling of that icon.
+The \`ButtonIconPosition\` property can be used to locate the icon on the left hand side of the text, the default is to the right hand side.
+Content projection is used as Canopy Icons need loading via the iconRegistry which enables tree shaking of icons.
+
+~~~html
+<button lg-button type="button" [ButtonIconPosition]="left" >
```

Comment:
I think the input name is wrong, could also drop the brackets...

```suggestion
<button lg-button type="button" iconPosition="left" >
```

Reply:
oh good spot, search and replace gone wrong!